modules/core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSRepositoryDescriptor.java

```diff
@@ -51,6 +51,18 @@ public Boolean isDefault() {
         return isDefault;
     }
 
+    @XNode("@isHeadless")
+    protected Boolean isHeadless;
+
+    /**
+     * Returns true if repository is headless.
+     *
+     * @since 11.2
+     */
+    public Boolean isHeadless() {
+        return isHeadless;
```

Comment:
Worth updating the Javadoc to state that this method can return `null`, and probably will in most of the cases.